.github/workflows/size.yaml

```diff
@@ -0,0 +1,40 @@
+name: Pull request size
+on: pull_request_target
+
+jobs:
+  size-label:
+    name: Size label
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Calculate size
+        run: |
+          #!/usr/bin/bash
+          set -euo pipefail
+
+          additions=$(curl -s "https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}" | jq '.additions')
+          deletions=$(curl -s "https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}" | jq '.deletions')
+          size=$((additions + deletions))
+          sizelabel=XS
+
+          if [ ${size} -gt 500 ]; then
+            sizelabel="XXL"
+          elif [ ${size} -gt 200 ]; then
+            sizelabel="XL"
+          elif [ ${size} -gt 100 ]; then
+            sizelabel="L"
+          elif [ ${size} -gt 50 ]; then
+            sizelabel="M"
+          elif [ ${size} -gt 30 ]; then
+            sizelabel="S"
+          else
```

Comment:
This `else` is not needed, I think.

Reply:
no, but I somehow felt that it makes it complete